complete/src/main/java/com/example/restservice/exceptions/SQInvalidRequestException.java

```diff
@@ -2,23 +2,31 @@
 
 import com.google.common.collect.ImmutableMap;
 
+import java.util.Map;
+
 public class SQInvalidRequestException extends SQException {
 
   private enum ReasonCode {
     QUEUE_NOT_FOUND,
     TOKEN_NOT_FOUND,
     QUEUE_NAME_ALREADY_EXISTS,
     TOKEN_NOT_NOTIFIABLE,
-    TOKEN_DELETED;
+    TOKEN_DELETED,
+    QUEUE_PAUSED,
+    QUEUE_DELETED,
+    QUEUE_DELETE_NOT_ALLOWED
   }
 
-  private final ImmutableMap<ReasonCode, String> message =
-      ImmutableMap.of(
+  private final Map<ReasonCode, String> message =
+      Map.of(
```

Comment:
just curious, why not ImmutableMap?

Reply:
Immutable map has a restriction of having max of five keys